src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -466,6 +466,41 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
         }
 
         [Fact]
+        public async Task GetAsync_ResponseWithSetCookieHeaders_AllCookiesRead()
+        {
+            // Cookie names/values, with one of the values deliberately over 128 characters,
+            // which is the size of WinHttpHandler's initial response header buffer size, to
```

Comment:
Then maybe this test really belongs in the System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs.

While we usually try to have most Http tests in System.Net.Http library so that we exercise both WinHttpHandler and CurlHandler, this test feels like it just belongs in WinHttpHandler library only.  In which case, you would adjust the test code to do a `new WinHttpHandler` etc.